src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface in the addressbook which creates a printable string representation of Name, Phone, Email, and Address.
```

Comment:
Is the header comment written in the correct style of a class header comment?

Don't mention the other classes which uses the interface in the header. You don't know who else will use this Interface in the future. Explain the purpose of it instead.